release.sh

```diff
@@ -15,32 +15,27 @@ if [[ ! $(command -v jq) ]]; then
 	exit 1
 fi
 
-if [[ ! -z $(git status -s) ]]; then
-    echo "Working tree not clean. Please re-run when the tree is clean."
-    exit
-fi
-
-local_rev=`git rev-parse HEAD`
-remote_rev=`curl -s https://api.github.com/repos/influxdata/vsflux/commits/master | jq -r .sha`
-if [[ $local_rev != $remote_rev ]]; then
-    echo "Local HEAD $local_rev is not the same as remote rev $remote_rev"
-    exit
-fi
+TEMPDIR=$(mktemp -d -t flux-release.XXXX)
+echo "Using fresh install in $TEMPDIR"
+cd $TEMPDIR
+git clone git@github.com:influxdata/vsflux.git > /dev/null 2>&1
```

Comment:
What is your opinion on using `set -e` to ensure that the script exits if a command like this one fails? You can also use a `trap` to delete the tempdir unconditionally on exit (but I guess if something goes wrong maybe you want to keep it around)

Reply:
Yeah, I like `set -e`, and am surprised I hadn't added it already. Thanks for the tip.